### DISCUSSION TITLE
Cannot create a swell storefront

### DISCUSSION THREAD

**Discussion by tommyharrisonlee** [swellstores/community#357] [Help] [answered]
Hello, I'm trying to customize the Swell hosted checkout page but in order to do so I have to create a Swell storefront. When trying to create a storefront I get stuck on the 'We are migrating your storefront configs. Please hold...' page.  Please help. Thanks

<img width="1377" alt="Screenshot 2023-08-27 at 6 35 27 PM" src="https://github.com/swellstores/community/assets/76571705/91983243-a9fb-4bb2-afbe-a7f7f39209eb">

<img width="1377" alt="Screenshot 2023-08-27 at 6 35 15 PM" src="https://github.com/swellstores/community/assets/76571705/335825ab-9fc9-4f7a-a892-87e26b9088db">

**Comment by logeshswell** [comment, marked as answer]
@tommylescott The issue should be fixed now. Can you check and confirm?
<img width="1512" alt="Screenshot 2023-08-28 at 1 26 02 PM" src="https://github.com/swellstores/community/assets/100766131/24c70dd9-6b89-48f0-a338-04db1195b9fa">

**Reply by tommyharrisonlee** [reply]
Thank you so much
